dev/bin/knative-installer.sh

```diff
@@ -19,6 +19,25 @@ function header_text {
   echo "$header$*$reset"
 }
 
+header_text "Initializing Knative Serving Core APIs"
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.5.0/serving-crds.yaml
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.5.0/serving-core.yaml
+
+header_text "Waiting for Knative Serving Core to become ready"
+kubectl wait deployment --all --timeout=900s --for=condition=Available -n knative-serving
+
+header_text "Initializing Kourier"
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/knative-v1.5.0/kourier.yaml
```

Comment:
Given that we have Istio, I expect kourier to not be necessary.
@r00ta can you please confirm?

Reply:
removed